geopandas/geoseries.py

```diff
@@ -1,9 +1,17 @@
 from functools import partial
 import json
+from packaging import version
```

Comment:
you can use the version-functionality from distutils (see eg how it is used elsewhere in geopandas here https://github.com/geopandas/geopandas/blob/3c66dece177478c14ee727abac338805b5f605ac/geopandas/io/file.py#L2), so then it is not needed to add an extra dependency

Reply:
Great !
I didn't kow about it, thank you !